src/components/Trace/TraceListDB/index.js

```diff
@@ -42,6 +50,18 @@ class TraceList extends PureComponent {
     );
   };
 
+  handleClick = (i) => {
+    const input = document.createElement('input');
+    input.value = i;
+    message.info('Copyed');
+    document.body.appendChild(input);
+    input.select();
+    if (document.execCommand('Copy')) {
+        document.execCommand('Copy');
```

Comment:
Why execute `document.execCommand('Copy')` twice?

Reply:
because it has compatibility requirements，so use **if**